Library/Formula/kafka.rb

```diff
@@ -26,6 +26,8 @@ def pour_bottle?
   end
 
   def install
+    ENV["_JAVA_OPTIONS"] = "-Duser.home=#{buildpath}"
```

Comment:
Still waiting on the problem we discussed in the other thread weeks ago and Tim reported upstream :cry: